### PR TITLE
TDI-38261:The variable of queue name return null on tAzureStorageOutput

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-38261
**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/319/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/319/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/325/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/325/)

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueWriter.java [hunk 1]
```diff
@@ -106,8 +106,10 @@ public void write(Object object) throws IOException {
                 CloudQueueMessage msg = new CloudQueueMessage(content);
                 msg.setMessageContent(content);
                 queue.addMessage(msg, ttl, visibility, null, null);
+                result.successCount++;
             }
         } catch (StorageException e) {
+        	result.rejectCount++;
```
Comment (gonchar-ivan) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueWriter.java]: please check formatting here
Seems, wrong indent here

Comment (pyzhou-talend) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueWriter.java]: It seems "Tab" does not show as 4 spaces in GitHub. The formatting is correct at my local eclipse. I will try to do some thing before merge. Thanks~

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/359/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/359/)

**Pull request merged**